bitnami/harbor/templates/jobservice/jobservice-cm.yaml

```
@@ -25,8 +25,8 @@ data:
       workers: {{ .Values.jobservice.maxJobWorkers }}
       backend: "redis"
       redis_pool:
-        redis_url: "{{ template "harbor.redisForJobservice" . }}"
-        namespace: "harbor_job_service_namespace"
+        redis_url: "{{ include "harbor.redisForJobservice" . }}"
```

Use "quote" instead:
```suggestion
        redis_url: {{ include "harbor.redisForJobservice" . | quote }}
```

---

Done